build/commands/lib/config.js

```diff
@@ -223,6 +223,7 @@ const Config = function () {
   this.brave_ai_chat_endpoint = getNPMConfig(['brave_ai_chat_endpoint']) || ''
   this.androidAabToApk = false
   this.enable_dangling_raw_ptr_checks = false
+  this.brave_ai_chat_feedback_hostname = getNPMConfig(['brave_ai_chat_feedback_hostname']) || ''
```

Comment:
@bridiver I think you're a codowner reviewer on this PR because of this file - can you take a look, please? This is a new endpoint from @porteron